setup.py

```diff
@@ -33,7 +33,7 @@
     packages=setuptools.find_packages(),
     python_requires=">=3.6",
     install_requires=[
-        "matplotlib>=2.0",
+        "matplotlib<=3.5.3",
```

Comment:
I think you do this to make the tests pass on Python 8. IMO, it is fine as temporary solution, but we should improve our tests sooner or later to make them resistant to the internal changes of matplotlib.

Reply:
I agree